internal/app/singularity/plugin_compile_linux.go

```diff
@@ -68,15 +68,75 @@ func CompilePlugin(sourceDir, destSif string) error {
 	return nil
 }
 
+// lookPath mostly copied from Go's exec.LookPath; modified to take a
+// path as an argument
+func lookPath(file, path string) (string, error) {
```

Comment:
To be honest, I'm really skeptical about this. I don't understand why we can't just use PATH